README.md

```diff
@@ -124,7 +124,7 @@ We have other projects tightly related to ArkScript, which aren't necessarily C+
 
 ### Contributing to the ArkScript standard library
 
-See [Coding guidelines](https://github.com/ArkScript-lang/Ark/wiki/Coding-guidelines#coding-in-ark) if you want to write ArkScript for the library (see folder `lib/std/`).
+See [Coding guidelines](https://github.com/ArkScript-lang/Ark/blob/dev/CONTRIBUTING.md#c-coding-guidelines) if you want to write ArkScript for the library (see folder `lib/std/`).
```

Comment:
This is actually the link to the C++ contributing guidelines, the ArkScript ones are below (on the page you linked)!

May I suggest renaming the section (Contributing to the ArkScript standard library) to `Coding guidelines for contributing` and linking the cpp contribution guidelines + the arkscript ones?

Something like a link if you want to contribute to cpp code, and another if you want to contribute to arkscript code

Reply:
I've commited an update.